controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,16 +364,55 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
+	var requeueForInfo bool
+
+	if instance.Spec.ReplicationState == replicationv1alpha1.Primary {
```

Comment:
(nit) When the state is secondary, we could drop the status update and mark time as a nil pointer, so that it does not carry a stale value.

Reply:
this seems to be done